packages/react-aria-components/src/Menu.tsx

```diff
@@ -241,7 +244,40 @@ function MenuInner<T extends object>({props, collection, menuRef: ref}: MenuInne
 const _Menu = /*#__PURE__*/ (forwardRef as forwardRefType)(Menu);
 export {_Menu as Menu};
 
-function MenuSection<T extends object>(props: SectionProps<T>, ref: ForwardedRef<HTMLElement>, section: Node<T>) {
+export interface MenuSectionProps<T> extends SectionProps<T>, MultipleSelection {}
+
+// A subclass of SelectionManager that forwards focus-related properties to the parent,
+// but has its own local selection state.
+class GroupSelectionManager extends SelectionManager {
```

Comment:
Nice, don't need to export it, so this can be specific to Menu and we can always add others as we need them